src/js/angular/rest/graphql.rest.service.js

```diff
@@ -65,23 +65,27 @@ function GraphqlRestService($http) {
     /**
      * Get the SHACL shape graphs for the given repository.
      * @param {string} repositoryId The repository ID.
-     * @returns {*} The SHACL shape graphs response.
+     * @returns {*|Promise<unknown>} The SHACL shape graphs response.
      */
     const getShaclShapeGraphs = (repositoryId) => {
         if (DEVELOPMENT) {
             return _mockBackend.getShaclGraphsMock(repositoryId);
         }
         return $http.get(`${REPOSITORIES_ENDPOINT}/${repositoryId}/manage/graphql/shacl_graphs`);
-        // FIXME: remove when the endpoint is present.
-        // return Promise.resolve({
-        //     data: {
-        //         shacl_graphs: [
-        //             // 'http://example.org/graph1',
-        //             // 'http://example.org/graph2',
-        //             // 'http://example.org/graph3'
-        //         ]
-        //     }
-        // });
+    }
+
+    /**
+     * Update the active state of the given endpoint in the backend.
+     * @param {string} repositoryId The repository ID.
+     * @param {string} endpointId The endpoint ID.
+     * @param {boolean} newActiveState The new active state.
+     * @returns {*|Promise<unknown>} The response from the backend.
+     */
+    const updateEndpointActiveState = (repositoryId, endpointId, newActiveState) => {
+        if (DEVELOPMENT) {
+            return _mockBackend.updateEndpointActiveStateMock(repositoryId, endpointId, newActiveState);
```

Comment:
The updateEndpointActiveStateMock function is not implemented.

Reply:
I know. Many of those are not implemented. When we finish with the implementation we are going to remove the mocks